src/components/tools/swot-analysis/steps/SWOTClassifyAlternativeActions/ModalClassifying/ModalClassifying.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {PureComponent} from "react";
+import {ClassificationValues,} from "../SWOTClassifyAlternativeActionsComponent";
+import {SelectClassificationModal} from "./SelectClassificationModal";
+
+import "./modal-classifying.scss";
+import {ClassifyingCard} from "./ClassifyingCard/ClassifyingCard";
+import {ClassificationAccordions} from "../NormalClassifying/ClassificationAccordions/ClassificationAccordions";
+import {NormalClassifyingProps} from "../NormalClassifying/NormalClassifying";
+
+
+interface ModalClassifyingProps extends NormalClassifyingProps {
+}
+
+interface ModalClassifyingState {
+    openClassificationModal: boolean
+    lastSelectedAction?: string
+}
+
+class ModalClassifying extends PureComponent<ModalClassifyingProps, ModalClassifyingState> {
+
+    constructor(props: ModalClassifyingProps | Readonly<ModalClassifyingProps>) {
+        super(props);
+
+        this.state = {
+            openClassificationModal: false
+        }
+    }
+
+    render() {
+        const classifications = this.props.classifications;
+
+        return (
+            <>
+                <ClassificationAccordions
+                    actions={this.props.actions}
+                    classifications={classifications}
+                    disabled={this.props.disabled}
+                    classificationController={this.props.classificationController}
+                    onClassificationClick={this.onOpenClassificationModalClick}
+                    cardElement={<ClassifyingCard/>}
```

Comment:
I think it would be better to set the disabled and click props here. So other command in ClassifyingCardList.